lib/filter_nb.c

```diff
@@ -1148,25 +1109,11 @@ static int lib_prefix_list_entry_action_modify(struct nb_cb_modify_args *args)
 	return NB_OK;
 }
 
-/*
- * XPath: /frr-filter:lib/prefix-list/entry/ipv4-prefix
- */
-static int
-lib_prefix_list_entry_ipv4_prefix_modify(struct nb_cb_modify_args *args)
+static int lib_prefix_list_entry_prefix_modify(struct nb_cb_modify_args *args)
 {
 	struct prefix_list_entry *ple;
 	struct prefix p;
 
-	if (args->event == NB_EV_VALIDATE) {
-		if (plist_is_dup_nb(args->dnode)) {
-			snprintf(args->errmsg, args->errmsg_len,
-				 "duplicated prefix list value: %s",
-				 yang_dnode_get_string(args->dnode, NULL));
-			return NB_ERR_VALIDATION;
-		}
-		return NB_OK;
-	}
-
 	if (args->event != NB_EV_APPLY)
```

Comment:
These two lines

```c
if (args->event != NB_EV_APPLY)
  return NB_OK;
```

duplicates what already happen in the caller.

Either:
1) Keep it here and remove from callers (which seems more unreadable)
2) Or remove here and keep on the callers (looks better, but has higher code line count)

Reply:
I have gone with option 1 and removed from the callers to reduce code lines.